theatre/studio/src/panels/SequenceEditorPanel/RightOverlay/Marker.tsx

```diff
@@ -0,0 +1,83 @@
+import type {Pointer} from '@theatre/dataverse'
+import {useVal} from '@theatre/react'
+import {pointerEventsAutoInNormalMode} from '@theatre/studio/css'
+import getStudio from '@theatre/studio/getStudio'
+import {lockedCursorCssPropName} from '@theatre/studio/uiComponents/PointerEventsHandler'
+import useContextMenu from '@theatre/studio/uiComponents/simpleContextMenu/useContextMenu'
+import useRefAndState from '@theatre/studio/utils/useRefAndState'
+import React from 'react'
+import styled from 'styled-components'
+import {attributeNameThatLocksFramestamp} from '@theatre/studio/panels/SequenceEditorPanel/FrameStampPositionProvider'
+import type {SequenceEditorPanelLayout} from '@theatre/studio/panels/SequenceEditorPanel/layout/layout'
+
+const Mark = styled.div<{left: number}>`
+  position: absolute;
+  height: 20px;
+  width: 8px;
+  background-color: white;
+  z-index: 1;
+  color: black;
+  overflow: hidden;
+  font-size: 10px;
+  left: ${({left}) => left}px;
+
+  ${pointerEventsAutoInNormalMode};
+
+  #pointer-root.draggingPositionInSequenceEditor & {
+    pointer-events: auto;
+    cursor: var(${lockedCursorCssPropName});
+  }
+
+  &:hover {
+    background-color: red;
+  }
+`
+
+const Marker: React.VFC<{
+  layoutP: Pointer<SequenceEditorPanelLayout>
+  marker: {time: number}
+}> = ({layoutP, marker}) => {
+  const layout = useVal(layoutP)
+  const clippedSpaceFromUnitSpace = layout.clippedSpace.fromUnitSpace
+
+  const [ref, node] = useRefAndState<HTMLDivElement | null>(null)
+
+  const [contextMenu] = useMarkerContextMenu(node, layout, marker.time)
+  return (
+    <>
+      {contextMenu}
+      <Mark
+        ref={ref}
+        {...{[attributeNameThatLocksFramestamp]: marker.time.toFixed(3)}}
+        data-pos={marker.time.toFixed(3)}
+        left={clippedSpaceFromUnitSpace(marker.time)}
```

Comment:
Hmm: The way this "left" is passed down feels a bit itchy to me. It literally means left with absolute for the styling. So, maybe just renaming it to `cssLeft` would help at least keep that awkwardness exposed a bit better in case there's something more interesting later (like a time -> left alignment css provider hook or something later, i dunno).

Reply:
Aria had a comment about this below too